app/models/container_deployment_authentication.rb

```diff
@@ -0,0 +1,5 @@
+class ContainerDeploymentAuthentication < ApplicationRecord
```

Comment:
All the encryption will be done on a diff pr.
This model have lots of columns that are unique to it and a lot of columns are missing on authentications table, would you rather add all those columns to the existing one?